plugins/afl-scoreboard/scroll_display.py

```
@@ -1,694 +1,347 @@
+
+            # Get scroll settings
+            scroll_settings = self._get_scroll_settings()
+            gap_between_games = scroll_settings.get("gap_between_games", 24)
+            show_separators = scroll_settings.get("show_league_separators", True)
+            game_card_width = scroll_settings.get("game_card_width", 128)
+
+            # Reuse the cached renderer (rebuilding it -- and reloading its
+            # fonts from disk -- on every call is wasteful); only rebuild if the
+            # card width changed, since GameRenderer bakes it in at construction.
+            if self._game_renderer is None or self._game_renderer_card_width != game_card_width:
+                self._game_renderer = GameRenderer(
```

Ruff (0.16.0)

[error] Undefined name `GameRenderer`

(F821)